manifests/stable/ptp.openshift.io_ptpoperatorconfigs.yaml

```diff
@@ -44,9 +44,9 @@ spec:
                   type: string
                 type: object
               plugins:
-                items:
-                  type: string
-                type: array
+                additionalProperties:
+                  x-kubernetes-preserve-unknown-fields: true
```

Comment:
Same here , need to check  if this will be an issue if we use conversion webhook.

Reply:
https://github.com/operator-framework/operator-lifecycle-manager/blob/c652154fd7eeb72a23f78856b2cbade775efad41/pkg/controller/install/webhook.go#L234

Looks like it has to be set to false for conversion webhook to work.